src/nvme/crc32.c

```diff
@@ -0,0 +1,100 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
The library is LGPL 2.1. So this is not compatible.

Reply:
there are alternatives:
https://github.com/rustyrussell/ccan/blob/master/ccan/crc32c/crc32c.h
http://web.archive.org/web/20080303102530/http://c.snippets.org/snip_lister.php?fname=crc_32.c

Reply:
Sadly, Rustys code is for crc32c (not crc32), and the crc source code from ccan also has GPLv2. So ccan doesn't help. The second implementation should be fine, though.

Reply:
But ever so slightly complicated to use. Switching to the freebsd implementation, which actually has a copy with a permissible license.